command/harness/destroy.go

```diff
@@ -24,19 +24,24 @@ func HandleDestroy(ctx context.Context, r *VMCleanupRequest, s store.StageOwnerS
 		return errors.NewBadRequestError("mandatory field 'stage_runtime_id' in the request body is empty")
 	}
 
+	logr := logrus.
+		WithField("stage_runtime_id", r.StageRuntimeID).
+		WithField("pool", r.PoolID).
+		WithField("api", "dlite:destroy")
+
+	logr.Traceln("starting the destroy process")
+
 	inst, err := poolManager.GetInstanceByStageID(ctx, r.PoolID, r.StageRuntimeID)
 	if err != nil {
 		return fmt.Errorf("cannot get the instance by tag: %w", err)
```

Comment:
add stage runtimeid here as well

Reply:
added this in the function itself: https://github.com/drone-runners/drone-runner-aws/pull/175/files#diff-9768b6394e00f9fc3eda342c26666650afc1ec6262709e84830d3abcc97ef214L106